source/extensions/extensions_build_config.bzl

```diff
@@ -124,11 +124,19 @@ EXTENSIONS = {
     #
     "envoy.matching.inputs.dynamic_metadata":           "//source/extensions/matching/http/metadata_input:metadata_input_lib",
 
+    #
+    # Transport Socket Matching Inputs
+    #
+    "envoy.matching.inputs.endpoint_metadata":     "//source/extensions/matching/common_inputs/transport_socket:config",
+    "envoy.matching.inputs.locality_metadata":     "//source/extensions/matching/common_inputs/transport_socket:config",
+    "envoy.matching.inputs.transport_socket_filter_state": "//source/extensions/matching/common_inputs/transport_socket:config",
+
     #
     # Matching actions
     #
 
     "envoy.matching.actions.format_string":             "//source/extensions/matching/actions/format_string:config",
+    "envoy.matching.action.transport_socket.name":      "//source/extensions/matching/common_inputs/transport_socket:config",
```

Comment:
You can probably just use a string here? https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/advanced/matching/matching_listener#id1